api/app.py

```diff
@@ -0,0 +1,125 @@
+import os, datetime
+from flask import Flask, jsonify, Response
+from flask_cognito import CognitoAuth
+from flask_cors import CORS
+
+try:
+    from config import api_config
+except Exception as e:
```

Comment:
Was this failing during development/debugging? This isnt very likely to fail in production, is it?

Reply:
No, ultra-unlikely to fail in production. 

All of this try/except business was added while I was debugging what turned out to be behavior in the API gateway. It's unhelpful to stay in, and I'm removing it - thank you for asking.